### ISSUE TITLE
Property or Method to retrieve the Xeokit (NPM) version from browser console.

### ISSUE THREAD

**Issue by blickfeld7** [xeokit/xeokit-sdk#431]
**Is this feature critical for your commercial enterprise?**
The feature is not critical for commercial enterprise, since there are other ways to track the version of xeokit with a deployment version. But it's definitely very cumbersome to find the exact version of xeokit in a deployed web app from a debugging perspective.

**Is your feature request related to a problem? Please describe.**
Currently there is no other way to know the version of xeokit, than looking into the code changes and compare them to the code in the github repo.

**Describe the solution you'd like**
I would like to retrieve the version number of xeokit (NPM version) from the browser console by calling a function or simply reading a constant value.
A public readonly property `version` on '@xeokit/xeokit-sdk/src/viewer/Viewer.js' would be an ideal solution.

**Describe alternatives you've considered**
An alternative to the readonly property is a method returning the version number.

**Comment by Amoki** [comment]
You can read `@xeokit/xeokit-sdk/package.json` and retrieve de version number from here it that fits your needs.

**Comment by blickfeld7** [comment]
Right, that's the place where to find the npm version usually. But in production react-scripts build doesn't include these sort of files. So once deployed it's not possible to see this file and thus get the npm version.
Sure one might add the file after build, but it's kind of hacky and there's a reason this file isn't included in the build.

**Comment by Amoki** [comment]
You can to it in your js/ts files

```javascript
import pkg from "@xeokit/xeokit-sdk/package.json";

const xeokitVersion = pkg.version;
```
Webpack and rollup handle it gracefully

**Comment by blickfeld7** [comment]
Ok got it, I need to add this to my code. Thanks your solution looks neat.
